rules/60-sensor.rules

```diff
@@ -7,4 +7,8 @@ SUBSYSTEM=="iio", KERNEL=="iio*", SUBSYSTEMS=="usb|i2c", \
   IMPORT{builtin}="hwdb 'sensor:modalias:$attr{modalias}:$attr{[dmi/id]modalias}'", \
   GOTO="sensor_end"
 
+DRIVERS=="cmpc_accel_v4", \
```

Comment:
> :acpi:$attr{hid}

This bit seems overly reliant on the hid name being the same as the one in the modalias. If you can show that it's always going to be the case, please mention it in the commit message, and that'll have to do. We'll see if we get other types of input sensors at some point.